src/api/schema/metrics/transform/generic.rs

```diff
@@ -24,4 +24,14 @@ impl GenericTransformMetrics {
     pub async fn processed_bytes_total(&self) -> Option<metrics::ProcessedBytesTotal> {
         self.0.processed_bytes_total()
     }
+
+    /// Events inputted for the current transform
```

Comment:
```suggestion
    /// Total incoming events for the current transform
```

I think "incoming" and "outgoing" read a little easier than "inputted" and "outputted".